storage.py

```diff
@@ -56,3 +57,25 @@ async def save_object(
 		)
 		obj._id = result.inserted_id
 		log.debug(f"Inserted {type(obj)} ({obj._id})")
+
+
+async def get_pokemon(id: ObjectId) -> Pokemon:
```

Comment:
(nit) Would prefer `load_pokemon` because `get` implies the operation is cheap, but we are actually hitting the database